wisp/wisp.php

```diff
@@ -597,3 +682,168 @@ function wisp_ClientArea(array $params) {
         // Ignore
     }
 }
+
+/* Utility Functions */
+function getNodes(array $params, int $loc_id) {
+    /*
+        Fetches and returns all nodes with a specific location_id
+    */
+    $filteredNodes = array();
+    $nodes = wisp_API($params, 'nodes/');
+    foreach($nodes['data'] as $key => $value) {
+        $node_id = $value['attributes']['id'];
+        if(!isset($loc_id) || $value['attributes']['location_id'] == $loc_id) {
+            array_push($filteredNodes, $node_id);
+        } else {
+            array_push($filteredNodes, $node_id);
```

Comment:
This seems redundant/test leftovers - it doesn't actually filter?

Reply:
Thanks for pointing that out - I was originally going to do some error handling in that else block but ended up inherently doing it in the main thread.
What I meant by the name "filteredNodes" is that they're "filtered" down to only the nodes that match the relevant location ID if that makes sense.

Updated 👍